src/ui.cpp

```diff
@@ -348,7 +349,7 @@ void uilist::filterlist()
 
 void uilist::inputfilter()
 {
-    input_context ctxt( input_category, keyboard_mode::keychar );
+    ctxt.set_category( input_category );
```

Comment:
This will make the filter context have actions registered in the list context, which may causes incorrect behavior,